OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -73,7 +73,7 @@ public string GetCursor(World world, CPos xy, MouseInput mi)
 			else
 			{
 				var frozen = world.ScreenMap.FrozenActorsAt(world.RenderPlayer, mi)
-					.Where(a => a.Info.HasTraitInfo<ITargetableInfo>() && !a.Footprint.All(world.ShroudObscures))
+					.Where(a => a.Info.HasTraitInfo<ITargetableInfo>() && world.RenderPlayer.CanViewActor(a.Actor))
```

Comment:
1. Needs a null check:
```
Exception of type `System.NullReferenceException`: Object reference not set to an instance of an object
  at OpenRA.Player.CanViewActor (OpenRA.Actor a) [0x00003] in /home/oliver/devel/openra/git/OpenRA.Game/Player.cs:157 
  at OpenRA.Orders.UnitOrderGenerator+<GetCursor>c__AnonStorey4.<>m__1 (OpenRA.Traits.FrozenActor a) [0x0001c] in /home/oliver/devel/openra/git/OpenRA.Game/Orders/UnitOrderGenerator.cs:76 
```

2. This prevents you from targeting frozen actors at all.